File: leetcode-submissions/1598_Crawler_Log_Folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int depth=0;
        for(string str:logs){
            if(str=="./"){
                continue;
            }else if(str=="../"){
                depth--;
                if(depth<0)depth=0;
            }else{
                depth++;
            }
        }
        return depth;
    }
};

File: leetcode-submissions/1598_Crawler_Log_Folder.ts
function minOperations(logs: string[]): number {
    let depth=0;
    for(const str of logs){
        if(str==="./"){
            continue;
        }else if(str==="../"){
            depth--;
        }else{
            depth++;
        }
    }
    return depth<0?0:depth ;
};